setup.md

```diff
@@ -27,16 +27,24 @@ In the lesson, you will find out how to access the data in this folder.
 > or [(KDE) Konsole](https://konsole.kde.org/)
 > or [xterm](https://en.wikipedia.org/wiki/Xterm),
 > which can be found via the applications menu or the search bar.
-> If your machine is set up to use something other than bash, you can run it by opening a terminal and typing `bash`.
+> 
+> To check if your machine is set up to use something other than Bash, type `echo $SHELL` in your terminal window.
+>
+> If your machine is set up to use something other than Bash, you can run it by opening a terminal and typing `bash`.
 >
 > ### macOS
-> For a Mac computer, the default Unix Shell is Bash,
-> and it is available via the Terminal Utilities program within your Applications folder.
+> For a Mac computer running Mac OS Mojave or earlier releases, the default Unix Shell is Bash.
```

Comment:
Can you please adjust naming to confirm with #1050

Reply:
Adjusted.